src/components/SubWheel.js

```diff
@@ -0,0 +1,29 @@
+const SubWheel = (props) => {
+    const buildLines = (subs) => {
+        return subs.map((line, index) => {
+          return <li key={index}><span>{line.sub.join(' ')}</span><span>{line.time}</span></li>
+        });
+    };
+
+    return (
+        <div className="subWheel">
+          <div className="preContextWrapper">
+            <ul className="preContext">
+              {buildLines(props.context.prev)}
```

Comment:
here you would use it like this

```suggestion
              <Lines subs={props.context.prev}/>
```